std/algorithm/setops.d

```diff
@@ -884,7 +885,8 @@ struct MultiwayMerge(alias less, RangeOfRanges)
             return;
         }
         // Put the popped range back in the heap
-        _heap.conditionalInsert(_ror.back) || assert(false);
+        assert(_heap.conditionalInsert(_ror.back), "Failed to insert item into"
```

Comment:
This insert will now be ommitted with -release

Reply:
thank you, my bad